app/views/layouts/application.html.erb

```diff
@@ -4,11 +4,11 @@
     <title>ToolsRUs</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
 
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload', defer: true %>
+    <script src="https://cdn.jsdelivr.net/npm/typed.js@2.0.11"></script>
```

Comment:
you don't need that because you already did `yarn add typed.js` it's one or the other.

Reply:
removed